test/api-permission.test.js

```diff
@@ -0,0 +1,22 @@
+describe('Shopify#apiPermission', () => {
+  'use strict';
+
+  const expect = require('chai').expect;
+
+  const common = require('./common');
+
+  const shopify = common.shopify;
+  const scope = common.scope;
+
+  afterEach(() => expect(scope.isDone()).to.be.true);
+
+  it('deletes an api permission', () => {
+    scope
+      .delete('/admin/api_permissions/current.json')
+      .reply(200, {});
```

Comment:
Is this the real thing that is returned? Can't see the response from the example in the doc.